.github/check_versions.py

```diff
@@ -8,11 +8,11 @@
 @contextmanager
 def read_conda_yaml(path):
     with open(path, "rb") as config_file:
-        # Toss out the first line that declares the version since its not supported YAML syntax
-        next(config_file)
+        # Toss out the first few line that declares the version and uses version in YAML since its not supported YAML syntax
+        for _ in range(5):
+            next(config_file)
         yield yaml.safe_load(config_file)
```

Comment:
The line `  version: {{ version }}` in our recipe is not considered proper yaml, so throws an error. Skipping the first 5 lines (instead of previously just skipping line where version is set to something) to pass our check :)